examples/billing_account/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Provider configuration
+ *****************************************/
+provider "google" {
+  version = "~> 2.7"
+}
+
+provider "google-beta" {
+  version = "~> 2.7"
+}
+
+/******************************************
+  Module billing_account_iam_binding calling
+ *****************************************/
+module "billing-account-iam" {
+  source = "../../modules/billing_accounts_iam/"
+
+  billing_account_ids = [var.billing_account_id]
+
+  mode = "additive"
```

Comment:
The [Caveats](https://github.com/terraform-google-modules/terraform-google-iam#referencing-valuesattributes-from-other-resources) section of the README indicates that dynamic members can only be used in authoritative mode. Can you add a test suite which exercises this example to ensure that it is functionally correct?

Reply:
Hi, I tested it myself and it worked fine, but suppose it's a bit risky to do testing against our Billing Account, the SA should be Billing Account Security Admin and some mistake in the PR can break a lot of things inside the testing org. 
@morgante what do you think?

Reply:
That's a fair point. I've open #80 to track the work.